tendrl/monitoring_integration/grafana/alert.py

```diff
@@ -0,0 +1,20 @@
+import maps
+
+from requests import get
+from tendrl.monitoring_integration.grafana import utils
+
+
+def get_alert(alert_id):
+    config = maps.NamedDict(NS.config.data)
```

Comment:
I understand this is to convert dict to obj type
But what is the advantage in this case?

Reply:
the advantage is json keys are only known by config class, if i use config as json then we need to hardcode the key to retrieve the value, so if any change in config file then we need to change wherever i hard coded. So in this case change in class in enough, no need to  worry about json key

Reply:
No but in this approach, if key name changes, then wouldn't the attribute name change correspondingly?

Reply:
mm yes, it may change. ok i will keep that as json